service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -503,7 +508,9 @@ private AddressInfoDto addressInfoDtoBuilder(Order order) {
             BagForUserDto bagDto = new BagForUserDto();
             bagDto.setCount(amountOfBags.get(bag.getId()));
             List<BagTranslation> translations = bag.getBagTranslations();
-            if (languageId == 2) {
+            if (translations.size() == 1) {
+                bagDto.setService(translations.get(0).getName());
+            } else if (languageId == 2) {
```

Comment:
why you don't use the binary operator OR ( || )
 if(translations.size() == 1) || languageId == 2) ?